pages/implementation/devs-101.mdx

```diff
@@ -1,152 +0,0 @@
----
```

Comment:
Huge! Didn't know we were deprecating this one!

Have we made sure we're still covering a high level overview of all our dev offerings somewhere, and how they compare (with links to deep-dive materials)? Have we made sure to setup a redirect from this page to that one?

May gut says we should keep this page, but boil it down to more high level, and link to the new juicy docs we've been building for Inspector/Codegen for the details.

Would that make sense?

Reply:
Yeah everything from here was moved to the Codegen overview and Inspector overview docs

Reply:
So maybe we'd need to write a new devs-101 doc

Reply:
Brought it back